src/main/java/frc/robot/commands/defaultCommands/IntakeDefaultCommand.java

```diff
@@ -0,0 +1,24 @@
+package frc.robot.commands.defaultCommands;
+
+import java.util.function.DoubleSupplier;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.IntakeSubsystem;
+
+public class IntakeDefaultCommand extends Command {
+
+    private final IntakeSubsystem intake;
+    private final DoubleSupplier bumper;
+
+    public IntakeDefaultCommand(IntakeSubsystem intake, DoubleSupplier bumper) {
```

Comment:
Do you want an intake bumper and an eject bumper? Or an eject button to just pop it out at full speed? We usually want to be able to run the intake in reverse on demand, in case something gets stuck.